[2024-12-17T05:45:24.558+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T05:45:24.566+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9863800249_2024-12-17T05-42-14-003772Z.check_if_callerid_is_present_in_calllogs_table scheduled__2024-12-17T05:42:14.003772+00:00 [queued]>
[2024-12-17T05:45:24.569+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9863800249_2024-12-17T05-42-14-003772Z.check_if_callerid_is_present_in_calllogs_table scheduled__2024-12-17T05:42:14.003772+00:00 [queued]>
[2024-12-17T05:45:24.569+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-17T05:45:24.575+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): check_if_callerid_is_present_in_calllogs_table> on 2024-12-17 05:42:14.003772+00:00
[2024-12-17T05:45:24.580+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=135677) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-17T05:45:24.580+0000] {standard_task_runner.py:72} INFO - Started process 135678 to run task
[2024-12-17T05:45:24.581+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9863800249_2024-12-17T05-42-14-003772Z', 'check_if_callerid_is_present_in_calllogs_table', 'scheduled__2024-12-17T05:42:14.003772+00:00', '--job-id', '559', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmpc8asjksz']
[2024-12-17T05:45:24.581+0000] {standard_task_runner.py:105} INFO - Job 559: Subtask check_if_callerid_is_present_in_calllogs_table
[2024-12-17T05:45:24.604+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9863800249_2024-12-17T05-42-14-003772Z.check_if_callerid_is_present_in_calllogs_table scheduled__2024-12-17T05:42:14.003772+00:00 [running]> on host 1868de310944
[2024-12-17T05:45:24.648+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9863800249_2024-12-17T05-42-14-003772Z' AIRFLOW_CTX_TASK_ID='check_if_callerid_is_present_in_calllogs_table' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T05:42:14.003772+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-17T05:42:14.003772+00:00'
[2024-12-17T05:45:24.648+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T05:45:24.655+0000] {base.py:84} INFO - Retrieving connection 'aicallagent'
[2024-12-17T05:45:24.660+0000] {sql.py:544} INFO - Running statement: SELECT * FROM call_logs WHERE caller_id = 'demo-scheduled-from-ui-global9e36b67b-2412-4707-adf2-cdacfd982842' limit 1;, parameters: None
[2024-12-17T05:45:24.662+0000] {sql.py:553} INFO - Rows affected: 1
[2024-12-17T05:45:24.663+0000] {python.py:240} INFO - Done. Returned value was: answered
[2024-12-17T05:45:24.675+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T05:45:24.676+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9863800249_2024-12-17T05-42-14-003772Z, task_id=check_if_callerid_is_present_in_calllogs_table, run_id=scheduled__2024-12-17T05:42:14.003772+00:00, execution_date=20241217T054214, start_date=20241217T054524, end_date=20241217T054524
[2024-12-17T05:45:24.713+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T05:45:24.726+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-17T05:45:24.727+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
